test/unit/validate_test.lua

```diff
@@ -616,10 +679,97 @@ local ok_cases = {
             },
         },
     },
+    ["integration_with_httpd_role_additional"] = {
+        cfg = { http = { { server = "additional" } } },
+        mocks = {
+            {
+                module = "config",
+                method = "get",
+                implementation = function (_, param)
+                    if param == "roles_cfg" then
+                        return {
+                            ['roles.httpd'] = httpd_config,
+                        }
+                    end
+                    return {}
+                end,
```

Comment:
It looks like the implementation the same for all mocks here and could be implemented as a separate function on the module-test-level to avoid copy-paste.

```suggestion
                implementation = config_get_return_httpd_config,
```

Reply:
Moved to the separate function.